Libraries/Animated/NativeAnimatedHelper.js

```diff
@@ -550,33 +550,14 @@ function transformDataType(value: number | string): number | string {
   }
 }
 
-module.exports = {
-  API,
-  isSupportedColorStyleProp,
-  isSupportedStyleProp,
-  isSupportedTransformProp,
-  isSupportedInterpolationParam,
-  addWhitelistedStyleProp,
-  addWhitelistedTransformProp,
-  addWhitelistedInterpolationParam,
-  validateStyles,
-  validateTransform,
-  validateInterpolation,
-  generateNewNodeTag,
-  generateNewAnimationId,
-  assertNativeAnimatedModule,
-  shouldUseNativeDriver,
-  transformDataType,
-  // $FlowExpectedError[unsafe-getters-setters] - unsafe getter lint suppresion
-  // $FlowExpectedError[missing-type-arg] - unsafe getter lint suppresion
-  get nativeEventEmitter(): NativeEventEmitter {
-    if (!nativeEventEmitter) {
-      nativeEventEmitter = new NativeEventEmitter(
-        // T88715063: NativeEventEmitter only used this parameter on iOS. Now it uses it on all platforms, so this code was modified automatically to preserve its behavior
-        // If you want to use the native module on other platforms, please remove this condition and test its behavior
-        Platform.OS !== 'ios' ? null : NativeAnimatedModule,
-      );
-    }
-    return nativeEventEmitter;
-  },
-};
+// $FlowExpectedError[missing-type-arg] - unsafe getter lint suppresion
+export function nativeEventEmitter(): NativeEventEmitter {
```

Comment:
This was previously a getter, now it's a function, which seems like an unsafe change

Reply:
You may be able to update the call-sites for this though, since this is an internal API: https://github.com/facebook/react-native/blob/2e649006f1a78539f5504bce79cbd51816679e13/Libraries/Animated/nodes/AnimatedNode.js#L127

Reply:
Got it, I'll revert this and keep it as a getter

Reply:
reverted